index.js

```diff
@@ -162,9 +163,10 @@ module.exports = function(version, _options) {
                 instruction = options.hooks.tokenizedInstruction(instruction);
             }
 
+            var waypoint = legIndex >= 0 && legIndex !== legCount - 1 ? this.ordinalize(language, legIndex + 1) : '';
```

Comment:
If it's the last of several waypoints, should we say "final destination"? The current code says only "destination".